### PR TITLE
Make Normalization Optional in Convolution

### LINKED ISSUE #939: Question: How to use even sized kernels with Monai Convolution block?
Hi All, I am working on rewriting [MC_GAN](https://github.com/HYOJINPARK/MC_GAN/blob/master/Model1/model.py) with the MONAI framework and have a question with the Monai Convolution block. Why are even number kernels not supported in the Monai Convolution block?

I am running into this problem with the block when my kernel size is an even number. The `monai.networks.layers.convutils.same_padding()` func called by `Convolution().__init__`  is throwing a NotImplementedError:
```python
# from monai.networks.layers.convutils.same_padding()

    if np.any((kernel_size_np - 1) * dilation % 2 == 1):
        raise NotImplementedError(
            f"Same padding not available for kernel_size={kernel_size_np} and dilation={dilation_np}."
        )

    padding_np = (kernel_size_np - 1) / 2 * dilation_np
```
Is there an implementation planned for this in the future? Why is there no manual padding option in the Convolution block constructor? 

I did some value testing and padding_np is an integer when k_size is odd, and ends in 0.5 when k_size is even. 

For reference these are the `nn.Sequential()` blocks:
```python
# Original PyTorch source I am replacing:
            torch.nn.Conv2d(in_chan, out_chan, kernel_size=4, stride=2, padding=1, bias=False)
            torch.nn.BatchNorm2d(out_chan)
            torch.nn.LeakyReLU(0.2, inplace=True)
# What I want to use: Monai Convolution Block
            Convolution(2, in_chan, out_chan, kernel_size=4, strides=1, bias=False, act=self.Act, norm=self.Norm)
# What I am using: Monai LayerFactory
            Conv["conv", 2](in_chan, out_chan, 4, 2, 1, bias=False),
            Norm["batch", 2](out_chan),
            Act["leakyrelu"](0.2, inplace=True),
```

Thank you for any help.

### PR DESCRIPTION
**Pull request by ericspod**
Fixes #939 (kind of).

### Description
Convolution can be specified with `norm=None` to omit normalization.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by Nic-Ma — APPROVED** [review]
Looks good to me.
Thanks.
